### DISCUSSION TITLE
Too Many Redirects

### DISCUSSION THREAD

**Discussion by ilbeygi** [postalserver/postal#1651] [Help with using Postal]
Hello

I install postal and configure according to the document. But ,when i want to load postal with URL, i see Too Many Redirects error !!!

![image](https://user-images.githubusercontent.com/23147213/138142447-ced469a1-9fc1-4804-afd5-590cb3167917.png)

thanks for yours help.

**Comment by willpower232** [comment]
Did you manage to resolve your nginx or caddy configuration?

**Reply by ilbeygi** [reply]
First i used caddy and get this problem, then i used nginx. that i see on port of 80 nginx welcome page and i can only see postal dashboard on port of 5000 .

**Reply by ilbeygi** [reply]
Also, when i want to use api for send email. I get this : UnauthenticatedFromAddress, Can yoh help me about this? thank you.

**Reply by willpower232** [reply]
If the api is working then it sounds like you have got a working website somewhere. To confirm, you should have caddy OR nginx, not both so if you configure one and disable the other then you should be okay.

`UnauthenticatedFromAddress` means you need to add the domain you are sending from to your Postal server and make sure that the credential you are using is from the same server.
